src/servers/src/http/event.rs

```diff
@@ -571,6 +573,29 @@ fn extract_pipeline_value_by_content_type(
             Deserializer::from_str(&payload).into_iter(),
             ignore_errors,
         )?,
+        ct if ct == *NDJSON_CONTENT_TYPE => {
+            let mut result = Vec::with_capacity(1000);
+            for (index, line) in payload.lines().enumerate() {
```

Comment:
can we add a test case when there is a line breaker in a string in json

Reply:
``` rust
    fn a() {
        let s1 = r#"
{"a": "a
"}
"#;
        let s2 = r#"
{"a": "a\n"}
"#;
        println!("s1: {:?}", s1.lines().collect::<Vec<&str>>());
        println!("s2: {:?}", s2.lines().collect::<Vec<&str>>());
    }
// output
s1: ["", "{\"a\": \"a", "\"}"]
s2: ["", "{\"a\": \"a\\n\"}"]
```

Unfortunately, if the line contains a __true__ line breaker, it will be treated as two lines. However if it's been escaped, it's fine.